lib/workers/repository/update/pr/automerge.ts

```diff
@@ -96,6 +96,12 @@ export async function checkAutoMerge(
         prAutomergeBlockReason: PrAutomergeBlockReason.DryRun,
       };
     }
+    if (GlobalConfig.get('dryRun') === 'lookup') {
+      return {
+        automerged: false,
+        prAutomergeBlockReason: PrAutomergeBlockReason.DryRun,
+      };
+    }
```

Comment:
```suggestion
```

It shouldn't get this far, and needs to be fixed earlier in the logic flow if it does